src/template.html

```diff
@@ -14,23 +14,22 @@ <h1>Leaderboard</h1>
     <div class="container">
       <div class="recent-score">
         <h2>Recent scores</h2>
-        <button>Refresh</button>
+        <button id="refreshBtn">Refresh</button>
       </div>
 
-      <div class="scores"></div>
+      <div class="scores"> </div>
     </div>
 
     <div class="container1">
       <h2>Add your score</h2>
       <form action="">
-        <input type="text" placeholder="Your name">
-        <input type="text" placeholder="Your score">
-        <button type="submit" id="submitbtn">Submit</button>
+        <input type="text" class="name" placeholder="Your name">
+        <input type="number" id="score" placeholder="Your score">
```

Comment:
[ OPTIONAL] 
- You can add proper validation to the name and score fields to ensure that a user cannot submit blank entries.

Reply:
@t-yanick , thank you so much, I will add it Right now ✋🏻